precompiles/author-mapping/Cargo.toml

```diff
@@ -11,7 +11,7 @@ num_enum = { version = "0.5.3", default-features = false }
 
 # Moonbeam
 pallet-author-mapping = { path = "../../pallets/author-mapping", default-features = false }
-precompile-utils = { path = "../utils", default-features = false }
+precompile-utils = { path = "../utils", features = ["testing"] }
```

Comment:
@nanocryk I had to make this change to run the tests locally

should I make this change to all the precompiles as well or how did you run tests locally?